docs/content/latest/yugabyte-cloud/cloud-network/cloud-vpc-intro.md

```diff
@@ -0,0 +1,75 @@
+
+![Peered VPCs](/images/yb-cloud/cloud-vpc-diagram.png)
+
+## Advantages
+
+Deploying your cluster in a VPC network has the following advantages:
+
+- Lower network latency. Traffic uses only internal addresses, which provides lower latency than connectivity that uses external addresses.
+- Better security. Your services are never exposed to the public Internet.
+- Lower data transfer costs. By staying in the provider's network, you won't have any Internet data transfer traffic. (Same region and cross region overages may still apply. Refer to [Data transfer costs](../../../cloud-admin/cloud-billing-costs/#data-transfer-costs).)
+
+## Pricing
+
+There's no additional charge for using a VPC. In most cases, using a VPC will reduce your data transfer costs.
+
+## Locating your VPC
+
+To avoid cross-region data transfer costs, deploy your VPC and cluster in the same region as the application VPC you are peering with.
+
+## Setting the CIDR and sizing your VPC
+
+A VPC is defined by a block of IP addresses, entered in [CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing). Because you cannot resize a VPC once it is created, you need to decide on an appropriate size before creating it. Calculate how many applications will be connecting to it, and estimate how that is expected to grow over time. You may want to create a large network to cover all contingencies, but a network that is over-sized can impact network performance. If your traffic experiences spikes, you will need to take that into account.
+
+When entering the range for your VPC in Yugabyte Cloud, the size of the network is determined by the prefix length (the number after the `/`). Yugabyte Cloud supports network sizes from `/24` to `/16` as shown in the following table.
+
+| Network Size (prefix length) | Number of Usable IP Addresses |
+| --- | --- | --- |
+| /16 | 65536 |
+| /17 | 32768 |
+| /18 | 16384 |
+| /19 | 8192 |
+| /20 | 4096 |
+| /21 | 2048 |
+| /22 | 1024 |
+| /23 | 512 |
+| /24 | 256 |
+
+You can use the following address ranges (per [RFC 1918](https://datatracker.ietf.org/doc/html/rfc1918)) for your VPCs:
+
+- 10.0.0.0        -   10.255.255.255  (10/8 prefix)
+- 172.16.0.0      -   172.31.255.255  (172.16/12 prefix)
+- 192.168.0.0     -   192.168.255.255 (192.168/16 prefix)
+
+Addresses cannot overlap with your other VPCs. Yugabyte Cloud warns you when you enter an overlapping range. You can calculate ranges beforehand using [IP Address Guide’s CIDR to IPv4 Conversion calculator](https://www.ipaddressguide.com/cidr).
+
+Yugabyte Cloud reserves the `10.21.0.0.16` range for internal operations.
+
+## Limitations
+
+- Yugabyte Cloud supports AWC and GCP for self-managed VPC peering.
+- You assign a VPC when you create a cluster. You cannot change VPCs after cluster creation.
+- You cannot change the size of your VPC once it is created.
+- VPC network ranges cannot overlap with the ranges of other networks in the same account.
```

Comment:
This is incorrect. They can create VPC with overlapping CIDRs. The only thing they need to know is that then ofcourse both these networks can't be peered with the same application network.